packages/web/docs/src/components/team-section/index.tsx

```diff
@@ -0,0 +1,166 @@
+import Image, { StaticImageData } from 'next/image';
+import { GlobeIcon } from '@radix-ui/react-icons';
+import { DiscordIcon, GitHubIcon, TwitterIcon } from '@theguild/components';
+import { cn } from '../../lib';
+import { ArrowIcon } from '../arrow-icon';
+import { CallToAction } from '../call-to-action';
+import { Heading } from '../heading';
+import dimaPhoto from './dima.webp';
+import noamPhoto from './noam.webp';
+import saihajPhoto from './saihaj.webp';
```

Comment:
I guess mine and Dima can be pulled from GH too?

Reply:
Theoretically, yes...
Practically, see the commit name.

https://github.com/kamilkisiela/graphql-hive/pull/5447/commits/d0f822126b749a7badc9100534245b4e8e2df81d

Reply:
ohh interesting okay!